### PR TITLE
Update import paths from github.com/go-gitea to code.gitea.io

### PR DESCRIPTION
**Pull request by strk**

### CODE REVIEW

**Comment by lunny** [comment]
LGTM

**Comment by strk** [comment]
There, Travis is fine now. Ready to merge (just needs one more lgtm)

**Comment by lunny** [comment]
Why this was merged with one lgtm?

**Comment by thibaultmeyer** [comment]
Because lgtm seems bugged or misconfigured. It use a simple `contains` operation to find the word `lgtm` without checking that the comment author is different from the PR author

File: cmd/serve.go [hunk 1]
```diff
@@ -14,12 +14,12 @@ import (
 	"time"
 
 	"github.com/Unknwon/com"
-	"github.com/go-gitea/git"
-	"github.com/go-gitea/gitea/models"
-	"github.com/go-gitea/gitea/modules/base"
-	"github.com/go-gitea/gitea/modules/httplib"
-	"github.com/go-gitea/gitea/modules/log"
-	"github.com/go-gitea/gitea/modules/setting"
+	"code.gitea.io/git"
```
Comment (tboerger) [inline cmd/serve.go:17]: This is currently not part of code.gitea.io

File: models/action.go [hunk 2]
```diff
@@ -16,12 +16,12 @@ import (
 	"github.com/Unknwon/com"
 	"github.com/go-xorm/xorm"
 
-	"github.com/go-gitea/git"
-	api "github.com/go-gitea/go-sdk/gitea"
+	"code.gitea.io/git"
+	api "code.gitea.io/go-sdk/gitea"
```
Comment (tboerger) [inline models/action.go:20]: This is totally wrong...

**Comment by tboerger** [comment]
I don't understand how travis could succeed on that. IMHO we should revert these changes

**Comment by ghost** [comment]
Everything just broke.

**Comment by thibaultmeyer** [comment]
It very strange i can `go build` without error but my IDE cant resolve any new dependency path.

**Comment by ghost** [comment]
I cannot `go get`. This is proof that the PR system needs a rework/something that prevents PRs from getting merged if they can't be built, not just on Travis CI but on multiple platforms.

`~ # go get -u -tags sqlite github.com/go-gitea/gitea
package code.gitea.io/gitea/cmd: unrecognized import path "code.gitea.io/gitea/cmd" (parse https://code.gitea.io/gitea/cmd?go-get=1: no go-import meta tags)
package code.gitea.io/gitea/modules/setting: unrecognized import path "code.gitea.io/gitea/modules/setting" (parse https://code.gitea.io/gitea/modules/setting?go-get=1: no go-import meta tags)`

**Comment by tboerger** [comment]
We need to revert or fix that asap. And we need a proper drone pipeline

**Comment by thibaultmeyer** [comment]
fix import would be better than revert commit cause other PRs has been merged since.

**Pull request merged**